app/src/main/res/raw/data.json

```diff
@@ -1738,5 +1738,2365 @@
         }}
+    ],
+    [
+    {"EN":{
+        "word": "cast",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "в ролях",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "cinema",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "кинотеатр",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "comedy",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "комедия",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "director",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "режиссер",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "documentary",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "документальный фильм",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "drama",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "драма",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "entertainment",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "развлечения",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "film",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "фильм",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "horror",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "ужасы",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "plot",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "сценарий",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "scene",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "сцена",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "screen",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "экран",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "star",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "звезда",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "stunt",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "трюк",
+        "topics": ["кино"]
+        }}
+    ],
+    [
+    {"EN":{
+        "word": "thriller",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "триллер",
+        "topics": ["кино"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "movie",
+        "topics": ["movies"]
+        }},
+    {"RU": {
+        "word": "фильм",
+        "topics": ["кино"]
+        }}
+    ],    
+	[
+    {"EN":{
+        "word": "january",
+        "topics": ["time"]
```

Comment:
probably better date?

Reply:
No, because "minute", for example, is not a date

Reply:
I understand, I mean to separate topics if it makes sense

Reply:
All this words about time